### ISSUE TITLE
Code crashes

### ISSUE THREAD

**Issue by aleix10kst** [Dragory/gulp-json-format#4]
Hello, I have a problem when formatting the json.
Here is the code.

`gulp.task('content', () => {
  remoteSrc(['en.json', 'es.json'], {
    //base: `${$baseUrl('development')}/content/`
    base: `http://localhost:3000/content/`
  }).pipe(jsonFormat(2)).pipe(gulp.dest('./src/content/i18n'));
});`

TypeError: file.pipe is not a function
    at /Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/gulp-json-format/index.js:23:8
    at wrappedMapper (/Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/map-stream/index.js:84:19)
    at Stream.stream.write (/Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/map-stream/index.js:96:21)
    at Stream.ondata (internal/streams/legacy.js:16:26)
    at emitOne (events.js:115:13)
    at Stream.emit (events.js:210:7)
    at queueData (/Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/map-stream/index.js:43:21)
    at next (/Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/map-stream/index.js:71:7)
    at /Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/map-stream/index.js:85:7
    at Request._callback (/Users/aleixcanet/WebstormProjects/arualbalCMS/node_modules/gulp-remote-src/index.js:67:21)

If I remove the first pipe(jsonFormat), it works...
any ideas?

**Comment by Dan503** [comment]
I'm guessing you are using Gulp 4? When Gulp released their latest update it had a large amount of breaking changes. No longer supporting `file.pipe()` was one of them.

I've created a pull request that fixes the issue but, since this hasn't been updated in 2 years I'm doubtful the maintainer will accept.

In the mean time, to fix the issue:

1. Save a local copy of `node_modules/gulp-json-format/index.js` into your source control.
2. Make the change that I made in [the pull request](https://github.com/Dragory/gulp-json-format/pull/5) to your local copy.
3. import/require your local copy instead of the node_modules copy

This should fix the issue for you.
